### PR TITLE
Update search-over-new-field.md

### PR DESCRIPTION
**Pull request by igbopie**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docs/how/search-over-new-field.md [hunk 1]
```diff
@@ -275,7 +275,7 @@ Response:
         "fullName": "Data Hub",
         "title": "CEO",
         "displayName": "Data Hub",
-        "email": "datahub@linkedin.com"
+        "email": "datahub@linkedin.com q
```
Comment (hangzhang925) [inline docs/how/search-over-new-field.md:278]: qqqqqqqqqqq?

Comment (igbopie) [reply docs/how/search-over-new-field.md:278]: Lol! hahaha

**Comment by mars-lan** [comment]
Why closing?

**Comment by igbopie** [comment]
not following the guidelines :P

**Pull request closed without merging**